### ISSUE TITLE
[MaterialToolbar] Cannot clear the navigation icon tint

### ISSUE THREAD

**Issue by Back4fire** [material-components/material-components-android#2678]
I can use MaterialToolbar. setNavigationIconTint(@ColorInt int navigationIconTint) to set the tint of the navigation icon. But it seems that there is no way to clear it. After looking through the source code, I think it will resolve the issue that simply changing the type of the parameter "navigationIconTint" from "int" to "Integer".

**Comment by dsn5ft** [comment]
Hi @Back4fire, I started looking into this but the tricky thing is that [DrawableCompat#setTint](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:core/core/src/main/java/androidx/core/graphics/drawable/DrawableCompat.java;l=136;drc=cacb6d54f7fea4cd372ac53d2f515dc358910eef) takes in an int.

However, there is a [DrawableCompat#setTintList](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:core/core/src/main/java/androidx/core/graphics/drawable/DrawableCompat.java;l=150;drc=cacb6d54f7fea4cd372ac53d2f515dc358910eef) method which takes a nullable ColorStateList. I tried using that and it seems to work; however, a key detail is that it will also clear any tint like `android:tint="?attr/colorControlNormal"` in the vector drawable xml, meaning the navigation icon drawable will take on the color of the `android:fillColor`.

So basically I can add a `materialToolbar.clearNavigationIconTint()` method to do the above, but I'm not sure how useful it will be.

What's your exact use case?

**Comment by Back4fire** [comment]
Hi, @dsn5ft, sorry for not replying in time. In my project, there is an activity that loading data from the server continuously. If the data is abnormal, the navigation icon ( and some other elements in UI ) will be changed to a special color to alert the user. If the data is normal, the navigation icon should be changed back to the normal color.
